src/modules/PublicLab.TitleModule.Related.js

```diff
@@ -81,15 +132,29 @@ module.exports = function relatedNodes(module) {
  
       if (module.options.suggestRelated) {
```

Comment:
This `if` block is present on the line 145 as well. They include 75% of the code in common. Can be refactored in a standalone function.